images/kube-webhook-certgen/OWNERS

```diff
@@ -1,10 +1,4 @@
-# See the OWNERS docs: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md
-
-approvers:
-- ingress-nginx-admins
-- ingress-nginx-maintainers
```

Comment:
i think we should keep at least the maintainers or drop the entire file

Reply:
But then we would also revoke reviewer privileges for @invidian. Is this intended?

Reply:
if the `ingress-nginx-maintainers` still works here then it is fine

Reply:
Yes, it should. I searched some older, already approved and merged PRs to check if the inheritance from the root OWNERS works, see above. According to my investigations and the docs it should work!